sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesOperation.cs

```diff
@@ -243,8 +243,7 @@ public virtual void Cancel(CancellationToken cancellationToken = default)
             scope.Start();
             try
             {
-                ResponseWithHeaders<TextAnalyticsCancelHealthJobHeaders> response = _serviceClient.CancelHealthJob(new Guid(_jobId), cancellationToken);
-                _operationInternal.RawResponse = response.GetRawResponse();
+                _serviceClient.CancelHealthJob(new Guid(_jobId), cancellationToken);
```

Comment:
Because `UpdateStateAsync` works with response from [`_serviceClient.HealthStatusAsync`](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesOperation.cs#L344)
and it is `UpdateStateAsync` that produces [correct state](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeHealthcareEntitiesOperation.cs#L372) for operation when it is canceled.

Reply:
Could you add this to the bug fixes in the Text Analytics changelog? This should be enough:
> In `AnalyzeHealthcareEntitiesOperation`, `Cancel` calls won't overwrite the response from `GetRawResponse` anymore.

Reply:
Done

Reply:
@AlexanderSher do you have the link to the issue you opened?